### PR TITLE
grammar

### PR DESCRIPTION
**Pull request by Pavel-Durov**
- SAME tokens for both functions and variables :muscle: 
- REMOVED annoying `;` postfix after each function call :muscle: 
- c-like grammar stolen from https://www.lysator.liu.se/c/ANSI-C-grammar-y.html :shushing_face:

Example of YAIWR program:
```
let x = 2;
let y = 3;

fun add (x, y){ 
    return x + y; 
}
println(add(x + 1, y + 1));

```

### CODE REVIEW

File: tests/lang/tests/arithmetics_mul.yaiwr [hunk 1]
```diff
@@ -1,4 +1,5 @@
 // Run-time:
-//   stdout: 27
+//    stdout: 27
+//    stderr:
```
Comment (vext01) [inline tests/lang/tests/arithmetics_mul.yaiwr:3]: So you are explicitly checking that stderr is empty?

Comment (Pavel-Durov) [reply tests/lang/tests/arithmetics_mul.yaiwr:3]: yes

File: src/calc.y [hunk 2]
```diff
@@ -133,3 +156,8 @@ fn parse_boolean(s: &str) -> Result<AstNode, ()> {
         }
     }
 }
+
+fn append(mut lhs: Vec<AstNode>, rhs: AstNode ) -> Result<Vec<AstNode>, ()>{
```
[the remark this answers is not in the document]
Comment (Pavel-Durov) [reply src/calc.y:160]: yeah I'll move it

**Comment by vext01** [comment]
LGTM, bar that `append()` thing.

**Comment by vext01** [comment]
Please squash.

**Comment by Pavel-Durov** [comment]
squashed https://github.com/softdevteam/pavel.yaiwr/pull/31/commits/14b068c61138c925f102c69fea09d13d731c863b

**Pull request merged**